### PR TITLE
[Feature] AV-3999 : Do not end on index error

### PR DESCRIPTION
**Pull request by YasheshSavani**
Ticket: https://20bn.atlassian.net/browse/AV-3999

Description:
This PR will add an update to `annotation.py` script to fix the issue of getting index error when all videos are annotated in a folder. Now, as an update after the last video's annotations submission it will redirect to the list of videos page.

### CODE REVIEW

File: scripts/tools/annotation/annotation.py [hunk 1]
```diff
@@ -11,9 +11,9 @@
 from sklearn.linear_model import LogisticRegression
 from joblib import dump, load
 
-dataset_path = '/home/amercier/code/20bn-realtimenet/fitness_tl_benchmark/'
+dataset_path = '/home/twentybn/Code/sense/dataset/'
```
Comment (antoinemrcr) [inline scripts/tools/annotation/annotation.py:14]: should not commit this change nor the next one

File: scripts/tools/annotation/prepare_annotation.py [hunk 2]
```diff
@@ -19,9 +19,9 @@
 # Create inference engine
 inference_engine = engine.InferenceEngine(feature_extractor, use_gpu=True)
 
-dataset_path = '/home/amercier/code/20bn-realtimenet/fitness_tl_benchmark/'
+dataset_path = '/home/twentybn/Code/sense/dataset/'
```
Comment (antoinemrcr) [inline scripts/tools/annotation/prepare_annotation.py:22]: do not commit this please

File: scripts/tools/annotation/annotation.py (hunk 1 above)
Comment (YasheshSavani) [reply scripts/tools/annotation/annotation.py:14]: ok

File: scripts/tools/annotation/prepare_annotation.py (hunk 2 above)
Comment (YasheshSavani) [reply scripts/tools/annotation/prepare_annotation.py:22]: ok

**Pull request merged**